frontend/src/components/Date/Date.jsx

```diff
@@ -0,0 +1,21 @@
+import './Date.scss';
+
+import React, { useEffect, useState } from 'react';
+
+const Day = () => {
```

Comment:
The naming of this component does not match the file name. It may be confusing. 
Consider renaming it to DateDisplay - Date is too general